mod/cornercutter/ModFeature/SpecialMods/PermanentFirstPerson/ApplyFirstPerson.cs

```diff
@@ -0,0 +1,59 @@
+﻿using cornercutter.ModLoader;
+using HarmonyLib;
+
+namespace cornercutter.ModFeature.SpecialMods.PermanentFirstPerson
+{
+    [HarmonyPatch(typeof(DungeonManager), nameof(DungeonManager.StartFloor))]
+    class PermanentFirstPerson
+    {
+        public static bool FirstPersonIsEnabled()
+        {
+            CutterConfig cornercutter = CutterConfig.Instance;
+            return cornercutter.CornercutterIsEnabled() && cornercutter.ModIsActive()
+                && cornercutter.IsSpecialModActive(ModNames.SpecialMod.FirstPerson);
```

Comment:
I feel like the `cornercutter.ModIsActive()` check should be done internally within `cornercutter.IsSpecialModActive()`, because the name makes it sound like it's also checking that the mod is active.